app/api/ada/lib/cardanoCrypto/cryptoWallet.js

```diff
@@ -37,7 +45,8 @@ export const isValidEnglishAdaPaperMnemonic = (
   phrase: string,
   numberOfWords: ?number = 27
 ) => {
-  const [unscrambled, unscrambledLen] = unscramblePaperAdaMnemonic(phrase, numberOfWords);
+  const fakePassword = numberOfWords === 21 ? 'xxx' : undefined;
```

Comment:
Any password will return some valid unscrambled mnemonic so we just pass a fake password to pass downstream validation